.github/workflows/reusable_export_pr_details.yml

```diff
@@ -84,7 +84,7 @@ jobs:
         with:
           github-token: ${{ secrets.token }}
           script: |
-            const script = require('.github/scripts/download_pr_artifact.js')
+            const script = require('.github/scripts/download_pr_artifact.cjs')
```

Comment:
When a node script is run at any level of a project, it looks for the nearest `package.json` file, which in this case is the one at the root.

Since that one has `"type": "module",` it sets the default mode to ESM for the script being run, which in this case makes it fail due to it being written using CJS (`require()` and such).

To tell Node.js to run this as CJS we had three options:
- Set the extension to `.cjs`
- Pass a flag at runtime (probably hard since we are running the script via a GitHub Action)
- Create a `package.json` file with `"type": "commonjs",` in the `.github/scripts/` directory

Alternatively we could've also rewrote the script as ESM, but in this case I decided not to and move forward.